egs/librispeech/ASR/pruned_transducer_stateless2/decode.py

```diff
@@ -53,6 +53,32 @@
     --beam 4 \
```

Comment:
if this is a floating-point beam, we should probably specify it at 4.0.  This might be a bit too low, leading to deletions at end of utterance.  You could try 8.

Reply:
Yes, I agree.

The above results are obtained using the command posted in https://github.com/k2-fsa/icefall/pull/420#issuecomment-1161349244

which indeed uses 8 (i.e., 8.0)

Reply:
OK, if 8.0 is not working well then try a much larger number, like 32.0.  We can even make it super large, and just rely on the max-states constraint.

Reply:
32.0 indeed helps. It reduces the WER to 2.82 from 3.43.

64.0 produces the same results as beam 32.0.